### DISCUSSION TITLE
Can this library be used independend of the arduino framework? With esp-idf?

### DISCUSSION THREAD

**Discussion by Gaai** [pschatzmann/arduino-audio-tools#1312] [Q&A] [answered]
Or do I need to add arduino as a component aswell then? Anyone any experience with this?

**Comment by pschatzmann** [comment, marked as answer]
Did you read the related wiki: [Using the Framework w/o Arduino](https://github.com/pschatzmann/arduino-audio-tools/wiki/Using-the-Framework-w-o-Arduino)?

**Reply by Gaai** [reply]
Thank you, that's awesome news. Google didn't get me to the wiki unfortunately, but I realize I should have looked at the wiki before asking here.
